google-cloud-bigquery/src/test/java/docs/javadsl/GoogleBigQuerySourceDoc.java

```diff
@@ -80,9 +82,12 @@ private static void example() {
 
   static Try<User> userFromJson(JsObject object) {
     return Try.apply(
-        () ->
-            new User(
-                object.fields().apply("uid").toString(), object.fields().apply("name").toString()));
+        () -> {
+          JsArray f = (JsArray) object.fields().apply("f");
+          String uid = ((JsString) f.elements().apply(0).asJsObject().fields().apply("v")).value();
+          String name = ((JsString) f.elements().apply(1).asJsObject().fields().apply("v")).value();
+          return new User(uid, name);
+        });
```

Comment:
I think we should consider providing a different default JSON implementation for the Java API, e.g. Jackson. Spray does not have a Java API and is painful to use as this example illustrates.

Reply:
I would recommend doing this in a follow up PR, if you're interested.